### PR TITLE
lld: bumped to 12.0.0

### PR DESCRIPTION
**Pull request by Ominitay**
Updated LLD for LLVM 12.0.0. Had to grab a file from `libunwind` due to a bug in the LLD standalone build. Not sure if it would be preferable to include just the `compact_unwind_encoding.h` file, or to go with getting it from `libunwind`.

## Existing package

- [x] I am the maintainer of this package.

### CODE REVIEW

File: community/lld/build [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 export DESTDIR="$1"
 
+mkdir -p include/mach-o
+mv libunwind/include/mach-o/compact_unwind_encoding.h include/mach-o/compact_unwind_encoding.h
```
Comment (dilyn-corner) [inline community/lld/build:6]: Could be `install -Dm644 libunwind/include/mach-o/compact_unwind_encoding.h include/mach-o/compact_unwind_encoding.h`

**Review by dilyn-corner — COMMENTED** [review]
If you just need the one file, it's probably just better to just include the one file. Then again, `libunwind.tar.xz` is pretty small. I'm fine with this.

**Comment by Ominitay** [comment]
> If you just need the one file, it's probably just better to just include the one file. Then again, `libunwind.tar.xz` is pretty small. I'm fine with this.

I think I *feel* more comfortable with grabbing the file from the tar file to be honest, since then we're not unnecessarily increasing the file size of the repo.

File: community/lld/build (hunk 1 above)
Comment (Ominitay) [reply community/lld/build:6]: Happy to change that now!

**Pull request merged**